frontend/src/components/GTerminalShortcut.vue

```diff
@@ -129,9 +129,9 @@ export default {
   ],
   setup () {
     const {
-      shootItem,
+      shootItem = null,
```

Comment:
```suggestion
      shootItem,
```

Reply:
this would break the context when coming from the button "GARDEN CLUSTER", since then `useShootItem` seems to return `null`.
```
GTerminalShortcuts.vue:135 Uncaught (in promise) TypeError: Cannot destructure property 'shootItem' of 'useShootItem(...)' as it is null.
```
this just tries to handle this gracefully so we stil lend up with `shootItem` as `null`.

there might be a better way to solve it in `useShootItem` to still return a result object instead of just `null`

Reply:
this seems to come from this default value here:
https://github.com/gardener/dashboard/blob/97c13571dc2199d32ca1dd3be31fe9a4867d21e8/frontend/src/composables/useShootItem.js#L264-L266

Reply:
ah I see, then how about adding a comment like the following to make it clear that the shoot-item can be absent in valid scenarios. WDYT?

```js
} = useShootItem() || {} // shoot-item is not provided in case of GardenTerminal
```

Reply:
definitly! i agree that its quite obscure and not super readable.
i will add the comments